aggregator/src/aggregator/aggregate_init_tests.rs

```diff
@@ -448,7 +448,7 @@ async fn aggregation_job_mutation_report_aggregations() {
 }
 
 #[tokio::test]
-async fn aggregation_job_init_two_round_vdaf_idempotence() {
+async fn aggregation_job_init_two_step_vdaf_idempotence() {
```

Comment:
We should probably keep this as-is. Poplar1 is two rounds, so this will still take one DAP step, just testing different code paths.
```suggestion
async fn aggregation_job_init_two_round_vdaf_idempotence() {
```